lib/fluent/plugin/in_prometheus.rb

```diff
@@ -111,16 +170,8 @@ def start
       end
 
       thread_create(:in_prometheus) do
```

Comment:
`in_prometheus_webrick` is better

Reply:
right. https://github.com/fluent/fluent-plugin-prometheus/pull/148/commits/49aa9c25b18384102dbb8e737fd7e9c165053cd7